tests/storage/outofprocess_test.py

```diff
@@ -49,6 +51,23 @@ def oop_cleanup():
     oop.stop()
 
 
+@pytest.fixture(
+    scope="function",
+    params=[
+        userstorage.PATHS["mount-512"],
+        userstorage.PATHS["mount-4k"],
```

Comment:
Testing 2 block sizes is important change that should be described in the commit message.

Add 4k block size can also be done later if it make this change harder. If everything works, we can keep the change here.